packages/fhir-converter/src/templates/cda/References/DiagnosticReport/assessment.hbs

```diff
@@ -29,8 +29,6 @@
     "resource":{
         "resourceType": "DiagnosticReport",
         "id":"{{ID}}",
-        "code":{{>DataType/CodeableConcept.hbs code=diagReport.code text=diagReport.title._ canBeUnknown=true}},
```

Comment:
You mentioned removing redundant elements - is this being added in another place

Reply:
Yea, those attributes are being attached to DiagReports in other places